src/styles/typing.css

```diff
@@ -0,0 +1,78 @@
+.sc-typing {
+  padding-top: 20px;
+  width: 300px;
+  margin: auto;
+}
+
+.sc-typing--circle {
+  width: 28px;
+  height: 22px;
+  padding: 5px 8px;
+  border-radius: 15px;
+  background-color: #e9ebee;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.sc-typing--dot {
+  background-color: #90949c;
+  -webkit-animation: typingAnimation 1.5s infinite ease-in-out;
+  animation: typingAnimation 1.5s infinite ease-in-out;
+  display: inline-block;
+  border-radius: 2px;
+  margin-right: 2px;
+  height: 4px;
+  width: 4px;
+}
+
+@-webkit-keyframes typingAnimation {
+  0% {
+    -webkit-transform: translateY(0px);
+    transform: translateY(0px)
+  }
+
+  28% {
+    -webkit-transform: translateY(-5px)
+  }
+
+  44% {
+    -webkit-transform: translateY(0px)
+  }
+}
+
+@keyframes typingAnimation {
+  0% {
+    -webkit-transform: translateY(0px);
+    transform: translateY(0px)
+  }
+
+  28% {
+    -webkit-transform: translateY(-5px)
+  }
+
+  44% {
+    -webkit-transform: translateY(0px)
+  }
+}
+
+.sc-typing--dot:nth-child(1) {
```

Comment:
can we name all these classes `typing-indicator` instead?

Reply:
Done